packages/core/src/server/helper.ts

```diff
@@ -279,7 +280,17 @@ export const getDevOptions = async ({
   return {
     devServerConfig: {
```

Comment:
can we avoiding merging the dev and server configs?

Reply:
done